apps/website/docs/pages/commands/overview.md

```diff
@@ -13,12 +13,14 @@ Sparo has four kinds of subcommands:
    - `sparo clone`
    - `sparo fetch`
    - `sparo pull`
```

Comment:
Looks like the website does not reflect the current `overview.md` file:
![image](https://github.com/tiktok/sparo/assets/4673363/9ecff6b2-d6ee-4de3-9644-201285ca3c05)

We need to remember to manually redeploy after updating the website, or else set up a GitHub Action for automated deployments.